src/wire/packet_manager.cpp

```diff
@@ -182,6 +210,16 @@ bool PacketManager::ProcessStartupPacket(InputPacket *pkt) {
   return true;
 }
 
+bool PacketManager::ProcessSSLRequestPacket(InputPacket *pkt) {
+  UNUSED(pkt);
```

Comment:
If `pkt` is not needed in this function, it is possible to remove it from arguments?